src/main/java/uk/ac/ebi/atlas/download/FileDownloadController.java

```diff
@@ -144,4 +153,58 @@ public FileDownloadController(ExperimentFileLocationService experimentFileLocati
             zipOutputStream.close();
         }
     }
+
+    @RequestMapping(value = "json/experiments/check/zip",
+            method = RequestMethod.GET,
```

Comment:
If you use `@GetMapping` you can remove the `method` here. You can do so for all endpoints in this file.

Reply:
Corrected.